Squeezer/src/main/res/layout/display_message.xml

```diff
@@ -1,41 +1,46 @@
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     android:id="@+id/display_message_container"
-    style="@style/DisplayMessage">
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:paddingStart="16dp"
```

Comment:
This line is not necessary

Reply:
What do you mean? Removing the padding surely makes a visual difference?

Reply:
That's the issue with these review tools, it's difficult to see which line is commented :-)
`paddingStart` is not necessary when you have `padding`. There should also be a warning in your IDE.

```
    android:paddingStart="16dp"
    android:padding="16dp"
```